Description: Use static tools on ppc64
 On ppc64, the shared binaries do not work either.

Index: klibc-2.0~rc2/usr/dash/Kbuild
===================================================================
--- klibc-2.0~rc2.orig/usr/dash/Kbuild	2012-02-11 19:04:51.000000000 +0000
+++ klibc-2.0~rc2/usr/dash/Kbuild	2012-02-11 19:04:51.000000000 +0000
@@ -93,7 +93,7 @@
 	$(Q):
 
 # Targets to install
-ifeq ($(ARCH),ia64)
+ifneq (,$(filter $(ARCH),ia64 ppc64))
 install-y := sh
 else
 install-y := sh.shared
Index: klibc-2.0~rc2/usr/kinit/fstype/Kbuild
===================================================================
--- klibc-2.0~rc2.orig/usr/kinit/fstype/Kbuild	2012-02-11 19:04:51.000000000 +0000
+++ klibc-2.0~rc2/usr/kinit/fstype/Kbuild	2012-02-11 19:04:51.000000000 +0000
@@ -22,7 +22,7 @@
 clean-dirs := static shared
 
 # install binary
-ifeq ($(ARCH),ia64)
+ifneq (,$(filter $(ARCH),ia64 ppc64))
 install-y := $(static-y)
 else
 install-y := $(shared-y)
Index: klibc-2.0~rc2/usr/kinit/ipconfig/Kbuild
===================================================================
--- klibc-2.0~rc2.orig/usr/kinit/ipconfig/Kbuild	2012-02-11 19:04:51.000000000 +0000
+++ klibc-2.0~rc2/usr/kinit/ipconfig/Kbuild	2012-02-11 19:04:51.000000000 +0000
@@ -28,7 +28,7 @@
 clean-dirs := static shared
 
 # install binary
-ifeq ($(ARCH),ia64)
+ifneq (,$(filter $(ARCH),ia64 ppc64))
 install-y := $(static-y)
 else
 install-y := $(shared-y)
Index: klibc-2.0~rc2/usr/kinit/nfsmount/Kbuild
===================================================================
--- klibc-2.0~rc2.orig/usr/kinit/nfsmount/Kbuild	2012-02-11 19:04:51.000000000 +0000
+++ klibc-2.0~rc2/usr/kinit/nfsmount/Kbuild	2012-02-11 19:04:51.000000000 +0000
@@ -24,7 +24,7 @@
 clean-dirs := static shared
 
 # Install binary
-ifeq ($(ARCH),ia64)
+ifneq (,$(filter $(ARCH),ia64 ppc64))
 install-y := $(static-y)
 else
 install-y := $(shared-y)
Index: klibc-2.0~rc2/usr/kinit/resume/Kbuild
===================================================================
--- klibc-2.0~rc2.orig/usr/kinit/resume/Kbuild	2012-02-11 19:04:51.000000000 +0000
+++ klibc-2.0~rc2/usr/kinit/resume/Kbuild	2012-02-11 19:04:51.000000000 +0000
@@ -27,7 +27,7 @@
 clean-dirs := static shared
 
 # install binary
-ifeq ($(ARCH),ia64)
+ifneq (,$(filter $(ARCH),ia64 ppc64))
 install-y := $(static-y)
 else
 install-y := $(shared-y)
Index: klibc-2.0~rc2/usr/kinit/run-init/Kbuild
===================================================================
--- klibc-2.0~rc2.orig/usr/kinit/run-init/Kbuild	2012-02-11 19:04:51.000000000 +0000
+++ klibc-2.0~rc2/usr/kinit/run-init/Kbuild	2012-02-11 19:04:51.000000000 +0000
@@ -26,7 +26,7 @@
 clean-dirs := static shared
 
 # install binary
-ifeq ($(ARCH),ia64)
+ifneq (,$(filter $(ARCH),ia64 ppc64))
 install-y := $(static-y)
 else
 install-y := $(shared-y)
Index: klibc-2.0~rc2/usr/utils/Kbuild
===================================================================
--- klibc-2.0~rc2.orig/usr/utils/Kbuild	2012-02-11 19:04:51.000000000 +0000
+++ klibc-2.0~rc2/usr/utils/Kbuild	2012-02-11 19:04:51.000000000 +0000
@@ -73,7 +73,7 @@
 clean-dirs := static shared
 
 # install only install the shared binaries
-ifeq ($(ARCH),ia64)
+ifneq (,$(filter $(ARCH),ia64 ppc64))
 install-y := $(static-y) static/reboot static/poweroff
 else
 install-y := $(shared-y) shared/reboot shared/poweroff
